_config.yml

```diff
@@ -14,6 +14,14 @@ url:  # the base hostname & protocol for your site
 baseurl: /al-folio # the subpath of your site, e.g. /blog/
 last_updated: # leave blank if you don't want to display last updated
 
+# -----------------------------------------------------------------------------
+# Open Graph
+# -----------------------------------------------------------------------------
+# Display links to the page with a preview object on social media.
+# To achieve this, change serve_og_meta to true and then provide the URL of the
+# preview image as the value of og_image.
+serve_og_meta: false # Include Open Graph meta tags in the HTML head
+og_image: # The Open Graph preview image
```

Comment:
If I understand correctly, `og_image` will be used for all pages of the website shared on social media, correct? This seems suboptimal for people who use al-folio for personal blogs, where each post is supposed to have a separate featured image.

Maybe there is a way to have `og_image` to be a page variable rather than the site variable? Or if the page does not have `og_image` specified, it could fall back to using `site.og_image`.

Reply:
> If I understand correctly, og_image will be used for all pages of the website shared on social media, correct? This seems suboptimal for people who use al-folio for personal blogs, where each post is supposed to have a separate featured image.

Yes, that's good thinking. Not my use case so didn't occur to me. Let me have a stab at that, it shouldn't be a problem.

Arguably, we should do the same for some of the other fields too (e.g. send the blog page title to the preview, instead of sending the entire site title)?